test/srm0-test.jl

```diff
@@ -0,0 +1,49 @@
+@plottest begin
```

Comment:
Can you change to `@test_skip @plottest`? Currently, PyPlot is causing CI issues, but I'd rather merge the working code than worry about that. Since we'll eventually move back to GR.